library/std/src/sys/pal/itron/abi.rs

```diff
@@ -1,4 +1,5 @@
 //! ABI for μITRON derivatives
+#![forbid(unsafe_op_in_unsafe_fn)]
```

Comment:
Everything under itron/ is already a sub-module of library/std/src/os/solid/mod.rs. so this should already be covered by the forbid above. The directory layout is a bit strange.

Reply:
Ah is that how that works? Okay!